waveorder/cli/apply_inverse_models.py

```diff
@@ -163,8 +160,7 @@ def birefringence_and_phase(
             yx_phase,
         ) = isotropic_thin_3d.apply_inverse_transfer_function(
             brightfield_3d,
-            absorption_transfer_function,
-            phase_transfer_function,
+            (U, S, Vh),
```

Comment:
Should this be considered a breaking change?

Reply:
Yes this is a breaking change. Do you think I need to make additional changes?

Reply:
(though these paths were broken before so I'm unsure)

Reply:
It's fine to have breaking changes. Just making sure that it is intended and we will write about it in the release note (of v3.0.0 I suppose).